cosmetics-web/app/models/notification.rb

```diff
@@ -108,7 +108,11 @@ def as_indexed_json(*)
       transitions from: :notification_file_imported, to: :draft_complete, guard: :formulation_present?
     end
 
-    event :submit_notification do
+    event :submit_notification, after: :cache_categories_for_csv! do
+      after do
+        cache_categories_for_csv!
+      end
+
```

Comment:
The `after` hook seems to be called twice when should only be called once, right?
```suggestion
    event :submit_notification, after: :cache_categories_for_csv! do   
```